app/models/user.rb

```diff
@@ -143,8 +143,8 @@ class User < ApplicationRecord
   validate :can_send_confirmation_email
   validate :update_rate_limit
 
-  alias_attribute :positive_reactions_count, :reactions_count
   alias_attribute :public_reactions_count, :reactions_count
+  alias_attribute :positive_reactions_count, :reactions_count
```

Comment:
I added this back in, in case any old caches are rely on it, like the cache key in `Reaction.cached_any_reactions_for?`. I don't think it needs it since I'm updating it in the same PR but maybe it does...?

Reply:
You can kill this, its only being used in the cache key, there is nothing in the cache that relies on it.